gexpect_test.go

```diff
@@ -188,6 +188,31 @@ func TestRegexFind(t *testing.T) {
 	}
 }
 
+func TestReadLine(t *testing.T) {
+	t.Logf("Testing ReadLine...")
+
+	child, err := Spawn("echo \"foo\nbar\"")
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	s, err := child.ReadLine()
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	if s != "foo\r" {
```

Comment:
The comment above was a bit too short - they end in \r\n and ReadLine reads pty until it finds \n, so it returns a line ending with \r.